hugo/content/downloads.md

```diff
@@ -74,7 +74,7 @@ document.getElementById('suggested-download').innerHTML = getPlatformContent(get
 
 ## Manual Download
 
-*Note: On Windows Microsoft Visual C++ 2015-2022 Redistributable (<a href="https://aka.ms/vs/17/release/vc_redist.x64.exe">x64 download</a> and <a href="https://aka.ms/vs/17/release/vc_redist.x86.exe">x86 download</a>) is required to be installed.*
+*Note: On Windows Microsoft Visual C++ 2022 Redistributable (<a href="https://aka.ms/vs/17/release/vc_redist.x64.exe">x64 download</a> and <a href="https://aka.ms/vs/17/release/vc_redist.x86.exe">x86 download</a>) is required to be installed.*
```

Comment:
I guess you can remove the reference to x86, right?

Reply:
Yeah

Reply:
```suggestion
*Note: On Windows Microsoft Visual C++ 2022 Redistributable (<a href="https://aka.ms/vs/17/release/vc_redist.x64.exe">x64 download</a>.*
```

Reply:
Right, I wanted to before askin in chat, but quickly prepared this after, and forgot :P

Reply:
Now I'm wondering whether that's true for the x86 overlay?

Reply:
No, we link the overlay library to the static CRT: https://github.com/mumble-voip/mumble/blob/7a6274f19e2d7c2ffb76b7ebcd22d0c1df9445d9/overlay/CMakeLists.txt#L146-L149